src/advanced-topics/optimizations.md

```diff
@@ -136,7 +136,9 @@ lto = true
 
 Further more it is possible to optimize size of `wasm` code.
 
-Info: [wasm-opt project](https://github.com/gonowa/wasm-opt)
+wasm-opt info: [binaryen project](https://github.com/WebAssembly/binaryen)
+
+Example how to use: [rustwasm code size optimalization](https://rustwasm.github.io/book/game-of-life/code-size.html)
```

Comment:
`optimalization` should be `optimization`
Instead of 'example how to use' I think it would be better to say `The Rust Wasm book features a section about reducing the size of WASM binaries`